Dockerfile

```diff
@@ -1,25 +1,28 @@
-FROM dmscid/epics-base
+FROM ubuntu:17.10
 
-RUN apk add --no-cache git gcc g++ linux-headers cmake make python py-pip python-dev bash
+RUN apt-get update && \
+    apt-get -y install cmake g++ git python-pip tzdata vim-common && \
+    apt-get -y autoremove && \
+    apt-get clean all
 
-RUN pip install --no-cache-dir conan
-RUN git clone -b master https://github.com/ess-dmsc/streaming-data-types.git
+RUN pip install --upgrade pip && \
```

Comment:
Needs to be `pip install --upgrade pip=9.0.1 && \`

There seems to be a breaking change in pip 10 which upsets docker.
I am sure this will be fixed soon...

Reply:
9.0.3 should be fine too